diff --git a/src/pneumoniaclassifier/train.py b/src/pneumoniaclassifier/train.py
index 9a9f52e..3e45fe3 100644
--- a/src/pneumoniaclassifier/train.py
+++ b/src/pneumoniaclassifier/train.py
@@ -207,8 +207,6 @@ def train(cfg: DictConfig) -> None:
 
     _init_wandb(cfg)
     if cfg.wandb.enabled and cfg.wandb.log_model:
-        import wandb
-
         wandb.watch(model, log="all", log_freq=100)
 
     global_step = 0
@@ -248,8 +246,6 @@ def train(cfg: DictConfig) -> None:
             if cfg.train.log_interval_steps > 0 and global_step % cfg.train.log_interval_steps == 0:
                 batch_acc = (preds == targets).float().mean().item()
                 if cfg.wandb.enabled:
-                    # import wandb
-
                     wandb.log(
                         {
                             "train/step_loss": loss.item(),
@@ -262,8 +258,6 @@ def train(cfg: DictConfig) -> None:
             if cfg.eval.interval_steps > 0 and global_step % cfg.eval.interval_steps == 0:
                 val_loss, val_acc = evaluate(model, val_loader, criterion, device)
                 if cfg.wandb.enabled:
-                    # import wandb
-
                     wandb.log(
                         {
                             "val/loss": val_loss,
@@ -282,8 +276,6 @@ def train(cfg: DictConfig) -> None:
             val_loss, val_acc = 0.0, 0.0
 
         if cfg.wandb.enabled:
-            # import wandb
-
             wandb.log(
                 {
                     "train/epoch_loss": epoch_loss,
diff --git a/wandb/debug-internal.log b/wandb/debug-internal.log
index 4610a7b..b46f77b 120000
--- a/wandb/debug-internal.log
+++ b/wandb/debug-internal.log
@@ -1 +1 @@
-run-20260112_160351-wec5ay4n/logs/debug-internal.log
\ No newline at end of file
+run-20260114_140910-zyemvze7/logs/debug-internal.log
\ No newline at end of file
diff --git a/wandb/debug.log b/wandb/debug.log
index 985f07c..1b257b0 120000
--- a/wandb/debug.log
+++ b/wandb/debug.log
@@ -1 +1 @@
-run-20260112_160351-wec5ay4n/logs/debug.log
\ No newline at end of file
+run-20260114_140910-zyemvze7/logs/debug.log
\ No newline at end of file
diff --git a/wandb/latest-run b/wandb/latest-run
index 004e19c..1383c69 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20260112_160351-wec5ay4n
\ No newline at end of file
+run-20260114_140910-zyemvze7
\ No newline at end of file
