diff --git a/configs/data/pneumonia.yaml b/configs/data/pneumonia.yaml
index 74066c9..47476eb 100644
--- a/configs/data/pneumonia.yaml
+++ b/configs/data/pneumonia.yaml
@@ -1,5 +1,5 @@
 _target_: pneumoniaclassifier.data.get_dataloaders
-data_dir: "data/chest_xray"
+data_dir: "/zhome/8a/1/224071/data/kaggle/chest-xray-pneumonia/chest_xray/"
 batch_size: 16
 num_workers: 2
 augment: true
\ No newline at end of file
diff --git a/src/pneumoniaclassifier/train.py b/src/pneumoniaclassifier/train.py
index 5c92094..9a9f52e 100644
--- a/src/pneumoniaclassifier/train.py
+++ b/src/pneumoniaclassifier/train.py
@@ -220,7 +220,7 @@ def train(cfg: DictConfig) -> None:
 
         train_progress = tqdm(
             train_loader,
-            desc=f"Epoch {epoch}/{train_config.train.epochs} [train]",
+            desc=f"Epoch {epoch}/{cfg.train.epochs} [train]",
             dynamic_ncols=True,
             leave=False,
         )
diff --git a/wandb/debug-internal.log b/wandb/debug-internal.log
index 7d36e06..4610a7b 120000
--- a/wandb/debug-internal.log
+++ b/wandb/debug-internal.log
@@ -1 +1 @@
-run-20260109_132201-yhhinz23/logs/debug-internal.log
\ No newline at end of file
+run-20260112_160351-wec5ay4n/logs/debug-internal.log
\ No newline at end of file
diff --git a/wandb/debug.log b/wandb/debug.log
index 1cc5827..985f07c 120000
--- a/wandb/debug.log
+++ b/wandb/debug.log
@@ -1 +1 @@
-run-20260109_132201-yhhinz23/logs/debug.log
\ No newline at end of file
+run-20260112_160351-wec5ay4n/logs/debug.log
\ No newline at end of file
diff --git a/wandb/latest-run b/wandb/latest-run
index 172c33c..004e19c 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20260109_132201-yhhinz23
\ No newline at end of file
+run-20260112_160351-wec5ay4n
\ No newline at end of file
